components/server/src/auth/personal-access-token.spec.ts

```diff
@@ -4,23 +4,20 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { suite, test } from "@testdeck/mocha";
 import { PersonalAccessToken } from "./bearer-authenticator";
 import { expect } from "chai";
+import { describe } from "mocha";
 
-@suite()
```

Comment:
just asking whether we have some convention about it?

I also started doing describe seeing that you do it with Sven, but I seen in another PR using suites, what is the right way?

Reply:
is it somehow related to VS Code tests integration?

Reply:
I agree we should put it down somewhere.

I prefer the functional way, because it's equivalent in funcationality, but avoids one abstraction layer (testdeck/mocha) which is a) not easier to use and b) did present challenges when upgrading in the past: It adds unnecessary baggage.

Reply:
ok, maybe at some point we remove another way then